usr/share/bash-completion/completions/kapacitor

```diff
@@ -165,6 +164,38 @@ _kapacitor()
     return 0
 }
 
+_kapacitor_files_add_postfix()
+{
+	#Add a slash (directories) or space (files except directories) after name
+	local c i=0
+	for c in "${COMPREPLY[@]}"; do
+		if [ -d "${c}" ]; then
+			COMPREPLY[i++]="${c}/";
+		else
+			COMPREPLY[i++]="${c} ";
+		fi
+	done
+        compopt -o nospace
```

Comment:
Isn't this the default? What is this doing? This is my first bash completion script so I don't really know what I am talking about..Thanks for improving it ;)